src/index.html

```diff
@@ -20,8 +21,393 @@
+              GADGETS
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              class="header__nav-link"
+              href="#tablets"
+            >
+              TABLETS
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              class="header__nav-link"
+              href="#photo"
+            >
+              PHOTO
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              class="header__nav-link"
+              href="#video"
+            >
+              VIDEO
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!-- #endregion -->
+
+    <!-- #region card -->
+    <main class="grid-container">
+      <article
+        class="card card--1"
+        data-qa="card"
+      >
+        <img
+          class="card__img"
+          src="../src/images/imac.jpeg"
+          alt="imac"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__text">Product code: 195434</p>
+        <div class="card__review">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__review-text">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-title">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+        <button
+          class="card__button"
+          data-qa="card-hover"
+        >
+          BUY
+        </button>
+      </article>
+
+      <article
+        class="card card--2"
+        data-qa="card"
+      >
+        <img
+          class="card__img"
+          src="../src/images/imac.jpeg"
+          alt="imac"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__text">Product code: 195434</p>
+        <div class="card__review">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__review-text">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-title">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+        <button
+          class="card__button"
+          data-qa="card-hover"
+        >
+          BUY
+        </button>
+      </article>
+
+      <article
+        class="card card--3"
```

Comment:
Remove it everywhere
```suggestion
        class="card"
```